src/main/resources/shared-security-context.xml

```diff
@@ -3,12 +3,15 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns:p="http://www.springframework.org/schema/p"
     xmlns:cache="http://www.springframework.org/schema/cache"
+	xmlns:security="http://www.springframework.org/schema/security"
     xsi:schemaLocation="
-                http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
-                http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache-3.2.xsd">
+                http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+                http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache.xsd
+				 http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security.xsd">
```

Comment:
Why was line 10 added?  It is unused.

Reply:
It should not be in this PR.  It is a reference from fixing a problem during an attempt to upgrade to spring-security 4.